contracts/exchange/test/core.ts

```diff
@@ -222,7 +222,24 @@ blockchainTests.resets('Exchange core', () => {
         };
         const privateKey = constants.TESTRPC_PRIVATE_KEYS[accounts.indexOf(makerAddress)];
         orderFactory = new OrderFactory(privateKey, defaultOrderParams);
-        fillOrderWrapper = new FillOrderWrapper(exchange, erc20Wrapper, erc721Wrapper, erc1155ProxyWrapper, provider);
+        fillOrderWrapper = new FillOrderWrapper(
+            exchange,
```

Comment:
This looks a little gnarly. Should these all be passed as an object?

Reply:
yeah on second thought I think I'll increase the number of parameter and skim off one layer of depth from the last two params

Reply:
actually I change my mind again 😬 
just gonna pull the last two params out into local variables